synapse/config/saml2_config.py

```diff
@@ -75,6 +75,7 @@ def default_config(self, config_dir_path, server_name, **kwargs):
         # override them.
         #
         #saml2_config:
+        #  enabled: true
```

Comment:
tbh I think this is redundant. The default is for 'enabled' to be considered `True`: the only reason that the config parser checks it is to avoid breaking people who have `enabled: false` in their config files.